Matlab/Miscellaneous.tmPreferences

```diff
@@ -34,21 +34,24 @@
 		<array>
 			<dict>
 				<key>name</key>
-				<string>TM_COMMENT_START</string>
+				<string>TM_COMMENT_START_2</string>
```

Comment:
Since this is *your* customization, *you* should take care of proper commenting functionality, i.e. override the `TM_COMMENT_*` variables for `source.octave source.matlab`. The default (or base syntaxes) that can be used alone should not be crippled by this.